scarb/tests/fmt.rs

```diff
@@ -58,6 +58,40 @@ fn simple_check_invalid() {
     assert_eq!(content, SIMPLE_ORIGINAL);
 }
 
+#[test]
+fn simple_emit_invalid() {
+    let t = build_temp_dir(SIMPLE_ORIGINAL);
+    Scarb::quick_snapbox()
+        .arg("fmt")
+        .arg("--emit")
+        .arg("stdout")
+        .arg("--no-color")
+        .current_dir(&t)
+        .assert()
+        .failure()
+        .stdout_matches(format!(
+            "{}:\n{}\n",
+            fs::canonicalize(t.child("src/lib.cairo"))
```

Comment:
```suggestion
            fsx::canonicalize(t.child("src/lib.cairo"))
```
Please use `fsx` instead, it's our internal extension of `fs`.